python/cudf/cudf/core/series.py

```diff
@@ -3381,8 +3473,7 @@ def __init__(self, series):
     @property  # type: ignore
     @_cudf_nvtx_annotate
     def year(self):
-        """
-        The year of the datetime.
+        """Get the year of the datetime.
```

Comment:
pandas doesn't use the "Get" term here or on other `dt` properties.
```suggestion
        """The year of the datetime.
```
https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.year.html

Reply:
I realize now that this is due to pydocstyle requiring a verb at the beginning. I wish it didn't require that for properties... oh well. Ignore this.

Reply:
Yeah this is an error code where I chose to prefer changing the docstring over adding noqas. If you change your mind and decide that you prefer the noqas let me know, I'm happy to change it back to that too.